charts/tidb-lightning/secret.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-storage-secret
+type: Opaque
+stringData:
+  rclone.conf: |
+    [s3]
+    type = s3
+    provider = AWS
+    env_auth = false
+    access_key_id = <my-access-key>
+    secret_access_key = <my-secret-key>
+    region = us-east-1
+
+    [ceph]
+    type = s3
+    provider = Ceph
+    env_auth = false
+    access_key_id = <my-access-key
+    secret_access_key = <my-secret-key>
+    endpoint = <ceph-object-store-endpoint>
+
+    [gcs]
+    type = google cloud storage
+    client_id =
+    client_secret =
+    token = {"AccessToken":"xxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx","RefreshToken":"x/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx_xxxxxxxxx","Expiry":"2014-07-17T20:49:14.929208288+01:00","Extra":null}
+    project_number = 12345678
+    object_acl = private
+    bucket_acl = private
```

Comment:
Why we need this file?

Reply:
This is a secret example to tell user how to create the secret.